src/reader/AlterMenu.js

```diff
@@ -42,33 +43,40 @@ export default function AlterMenu({
 
     return word.source;
   }
-
   return (
     <AlterMenuSC ref={refToAlterMenu}>
-      {word.alternatives.map((each) => (
-        <div
-          key={each.translation}
-          onClick={(e) =>
-            selectAlternative(each.translation, shortenSource(each))
-          }
-          className="additionalTrans"
-        >
-          {each.translation}
-          <div style={{ fontSize: 9, color: zeeguuDarkOrange }}>
-            {shortenSource(each)}
+      {word.alternatives === undefined ? (
+        <LoadingAnimation
+          specificStyle={{ height: "1em", margin: "1em 3em" }}
+          delay={100}
```

Comment:
maybe it should be 200ms? e.g. https://medium.com/@kaleemkhan/what-is-the-ideal-waiting-time-before-you-show-loading-indicator-cf094528faec